### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jutoft**
added description on how to extract Z-Stack_Home

### CODE REVIEW

File: org.openhab.binding.zigbee/README.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ This is the Texas Instruments ZNP stack. The thing type is ```coordinator_cc2531
 The CC2531 USB dongle must be flashed with the correct firmware in order to work with this binding.
 The file can be downloaded from TI website archives (http://www.ti.com/tool/z-stack-archive) as part
 of the `Z-STACK-HOME v.1.2.2a` package.
-The file name is `CC2531ZNP-Pro-Secure_Standard.hex` and its sha256 is `3cc5dc571ef0f49e3f42c6c2ca076d6f8fef33a945c71e6f951b839ba0599d3c`.
+Wine can be used to run the downloaded file extract the file.
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md]: I would suggest to remove the point about Wine as it is system specific.

File: org.openhab.binding.zigbee/README.md [hunk 2]
```diff
@@ -70,7 +70,8 @@ This is the Texas Instruments ZNP stack. The thing type is ```coordinator_cc2531
...
+The file name is `CC2531ZNP-Pro-Secure_Standard.hex` path `.wine/drive_c/Texas Instruments/Z-Stack Home 1.2.2a.44539/Projects/zstack/ZAP/ZNP-HexFiles/CC2531/CC2531ZNP-Pro-Secure_Standard.hex` and its sha256 is `3cc5dc571ef0f49e3f42c6c2ca076d6f8fef33a945c71e6f951b839ba0599d3c`.
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/README.md]: The folder shown here is system specific, and I think should really reference the location in the EXE file without the specific location in which it is extracted.

**Comment by t-8ch** [comment]
Maybe it could be documented that certain devices (Xiaomi for example) need the LinkKeyJoin firmware.

**Comment by cdjackson** [comment]
Closing this as it's been open for months. Please feel free to reopen if the updates are made.

**Pull request closed without merging**